lib/rules/resolve-async.js

```diff
@@ -12,6 +12,7 @@ const utils = require("../utils");
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I think this one should be a "problem", since unresolved async controls (whether due to `QUnit.stop`, `QUnit.asyncTest`, or `assert.async`) result in the test hanging.

Reply:
Sounds good, fixed!